libafl/src/executors/hooks/inprocess.rs

```diff
@@ -195,7 +195,17 @@ impl<I, S> HasTimeout for InProcessHooks<I, S> {
 impl<I, S> ExecutorHook<I, S> for InProcessHooks<I, S> {
     fn init(&mut self, _state: &mut S) {}
     /// Call before running a target.
-    fn pre_exec(&mut self, _state: &mut S, _input: &I) {
+    fn pre_exec(&mut self, _state: &mut S, _input: &I) {}
+
+    /// Call after running a target.
+    fn post_exec(&mut self, _state: &mut S, _input: &I) {}
+}
+
+impl<I, S> InProcessHooks<I, S> {
+    /// Setting up crash handler
+    // # Safety
+    // Don't call this from multiple threads. It is accesing a global variable.
+    pub(crate) unsafe fn enter_target_hooks(&mut self) {
```

Comment:
Why not name this guy `setup_crash_handlers` then?
The safety comment should have three `///`